nautobot/ipam/models.py

```diff
@@ -1065,13 +1069,17 @@ def _deconstruct_address(self, address):
     natural_key_field_names = ["parent__namespace", "host"]
 
     def _get_closest_parent(self):
+        if self._closest_parent:
+            return self._closest_parent
```

Comment:
Is this just a performance optimization or was there a functional need for this caching? If it's needed, I feel like we need logic to automatically clear this cache, e.g. when changing `self._namespace` should automatically clear `self._closest_parent` to avoid inconsistent data. Might be simpler to just leave it out if it's not actually required.

Reply:
Okay, Illd just remove it, and add a TODO